custom-commands/smart-room-lite/README.md

```diff
@@ -0,0 +1,11 @@
+# Smart Room Lite - A Custom Commands application
+
+## Purpose
+This application is built with the goal of being used as a quick-start for documentation hosted on docs.microsoft.com
+Alternatively, it can also be used by the users to verify their changes, in case they have difficult following through the documentation.
+
+Here, we have tried to use most of the features provide by the Custom Commands platform to build a simple application.
+
+## Links
+
+For usage, refer to official [documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/custom-commands)
```

Comment:
This link does not provide any usage information. What scenarios does this app provide? Example sentences?

Reply:
Yeah, the link shuold take you right to the quick start page which talks about building this demo

Reply:
I have changed ref to point to the currently hosted documentation. We will need to revisit this, once we have the new documentation changes in -- cause the link address would change at that point.

Reply:
@chschrae - I have modified the readme file as per your recommendation to include domain and example sentences. Can you please take a look?

Reply:
Resolving as discussed offline